README.md

```diff
@@ -60,10 +60,12 @@ NOT REQUIRED:
     - Description for the launch
 --variable RP_LAUNCH_ATTRIBUTES:"RF tag_name:tag_value"
     - Space-separated list of tags/attributes for the launch
---variable RP_TEST_ATTRIBUTES:"Long"
+--variable RP_TEST_ATTRIBUTES:"key:value"
```

Comment:
Please, add more tag examples. Not only ":" separated.

Reply:
Done